pkgs/tools/inputmethods/fcitx5/fcitx5-gtk.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, fetchurl
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, fcitx5
+, gobject-introspection
+, libxkbcommon
+, gtk2
```

Comment:
It may be useful to set it to optional or split it to two packages like fcitx5-gtk{2,3}
since some of us don't want to use gtk2 ( gtk2 is already EOL, although some apps still use it...

Reply:
I've made gtk2 support optional.